src/net_processing.cpp

```diff
@@ -1479,7 +1479,23 @@ void RelayTransaction(const uint256& txid, const uint256& wtxid, const CConnman&
     });
 }
 
-static void RelayAddress(const CAddress& addr, bool fReachable, const CConnman& connman)
+/**
+ * Relay (gossip) an address to a few randomly chosen nodes.
+ * We choose the same nodes within a given 24h window (if the list of connected
+ * nodes do not change) and we don't relay to nodes that already know an
+ * address. So within 24h we will likely relay a given address once. This is to
+ * prevent someone to unjustly give their address better propagation by sending
+ * it to us repeatedly.
+ */
+static void RelayAddress(
+    /** [in] The peer that sent us the address. We don't want to try to relay it back. */
+    const CNode* originator,
+    /** [in] Address to relay. */
+    const CAddress& addr,
+    /** [in] Whether the address' network is reachable. We relay unreachable addresses less. */
+    bool fReachable,
+    /** [in] Connection manager to choose nodes to relay to. */
```

Comment:
I think the Doxygen style in developer-notes.md might be easier to read.

Line 1485: s/do/does/

Reply:
The guidelines already [say](https://github.com/bitcoin/bitcoin/blame/86f20071931b803b5f26ed8f685d98d4919fb7a7/doc/developer-notes.md#L140)

> Use Doxygen-compatible comment blocks for functions

and the above is (was) doxygen-compatible, so I don't think there is anything to update in `developer-notes.md`.

The used inline comments have numerous advantages over `@param` which I will summarize and try to get some feedback to eventually get some explicit "allow that in new code".

Anyway, no need to block this PR with that, so I have changed it to use `@param`.